deploy/packer/base/scripts/packages.sh

```diff
@@ -127,7 +127,14 @@ sudo sh -c 'echo "options v4l2loopback devices=100" > /etc/modprobe.d/v4l2loopba
 
 # Install Audio loopback devices
 echo "snd-aloop" | sudo tee --append /etc/modules
-sudo sh -c 'echo "options snd-aloop enable=1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1 index=0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29" > /etc/modprobe.d/snd-aloop.conf'
+sudo sh -c 'echo "options snd-aloop enable=1,1,1,1,1,1,1,1 index=0,1,2,3,4,5,6,7" > /etc/modprobe.d/snd-aloop.conf'
+
+sudo bash -c 'cat > /etc/rc.local <<EOF
+#!/bin/sh -e
+modprobe snd-aloop
+exit 0
+EOF'
+sudo chmod +x /etc/rc.local
```

Comment:
This should be done in an `/etc/modprobe.d/` file.

Reply:
Err, forgot to add a comment here. I don't know why but in this new kernel putting `snd-aloop` inside `/etc/modules` doesn't load it at boot time, with no trace in `dmesg`. I put the `modprobe` in this file to make sure the module is correctly loaded.